src/Axiom/DoubleNegationElimination.agda

```diff
@@ -0,0 +1,34 @@
+------------------------------------------------------------------------
+-- The Agda standard library
+--
+-- Results concerning double negation elimination.
+------------------------------------------------------------------------
+
+{-# OPTIONS --without-K --safe #-}
+
+module Axiom.DoubleNegationElimination where
+
+open import Axiom.ExcludedMiddle
+open import Level
+open import Relation.Nullary
+open import Relation.Nullary.Negation
+
+------------------------------------------------------------------------
+-- Definition
+
+-- The classical statement of double negation elimination says that
+-- if a property is not not true then it is true.
+
+DoubleNegationElimination : (ℓ : Level) → Set (suc ℓ)
+DoubleNegationElimination ℓ = {P : Set ℓ} → ¬ ¬ P → P
+
```

Comment:
I'd state it for a specific type as for `UIP`. This way we can
include proofs that `Decidable P → DoubleNegationElimination P`.

Reply:
And also `DoubleNegationElimination (Dec P) → Decidable P`

Reply:
Hmm... that would be non-backwards compatible and would break the equivalence with `ExcludedMiddle`?

Reply:
I think it's worth it: these 'classical' results are useful in practice for
types with a decision procedure. Having the lemmas stated at a specific type
means they are more reusable.

Reply:
Hmm now that I think about it we already have 'Stable' in 'Relation.Nullary.Negation' for what your suggesting. To state that either excluded middle or double negation elimination holds for a specific type is simply to say that that 'Dec A' is true for 'A'. Isn't the whole point of excluded middle is that it says that it's true for any 'A', no?

Reply:
Apologies, given that [Stable](https://github.com/agda/agda-stdlib/blob/master/src/Relation/Nullary/Negation.agda) is exactly what you want, I'm going to merge this in as is.